tests/bin/cli/update-reconciliation.test.js

```
@@ -0,0 +1,94 @@
+const fs = require("fs");
```

ESLint

[error] 'fs' is assigned a value but never used. Allowed unused vars must match /^_/u.

(no-unused-vars)